Source/WebKit/GPUProcess/graphics/RemoteGraphicsContextGLFunctionsGenerated.h

```diff
@@ -1641,13 +1641,19 @@
 #if ENABLE(WEBXR)
     void createExternalImage(uint32_t name, WebCore::GraphicsContextGL::ExternalImageSource&& arg0, uint32_t internalFormat, int32_t layer)
     {
+        if (!webXRPromptAccepted())
```

Comment:
This needs to come from the `./Tools/Scripts/generate-gpup-webgl` script, I will modify that instead

Reply:
Where is this change?

I believe you can update the prototypes in `RemoteGraphicsContextGLProxy.h` from:
```
    GCGLExternalImage createExternalImage(WebCore::GraphicsContextGL::ExternalImageSource&&, GCGLenum internalFormat, GCGLint layer) IPC_MESSAGE_ATTRIBUTE(EnabledIf='webXREnabled()') final;
```
to
```
    GCGLExternalImage createExternalImage(WebCore::GraphicsContextGL::ExternalImageSource&&, GCGLenum internalFormat, GCGLint layer) IPC_MESSAGE_ATTRIBUTE(EnabledIf='webXRPromptedAccepted()') final;
```
and the tooling will generate the check for you.

Reply:
The change was reverted after I updated `./Tools/Scripts/generate-gpup-webgl`

Reply:
Why is the change to `generate-gpu-webgl` not part of this patch?

Reply:
Oh forgot to update, the changes ended up being limited to RemoteGraphicsContextGL.messages.in https://github.com/WebKit/WebKit/pull/30612/files#r1672818808